src/appforms/src/core/020-utils02webcam.js

```diff
@@ -24,10 +24,18 @@ appForm.utils = function (module) {
     checkEnv();
     _html5Camera(params, cb);
   }
+
   function cancelHtml5Camera() {
-    if (localMediaStream) {
-      localMediaStream.stop();
-      localMediaStream = null;
+    if (localMediaStream){
+      if (localMediaStream.stop) {
+        localMediaStream.stop();
+        localMediaStream = null;
+      } else{
+        var tracks = localMediaStream.getTracks();
```

Comment:
This implementation is just copy/paste from official chrome implementation.
In documentation they saying that getTracks must return something. In some really corner cases it will return empty array.

http://www.w3.org/TR/mediacapture-streams/#widl-MediaStream-getTracks-sequence-MediaStreamTrack